doc/CHANGELOG.md

```diff
@@ -2,15 +2,25 @@
 
 ### Added
 
-- Open recent files. (#527)
-- Reveal the executable file in file manager. (#537)
+-   Open recent files. (#527)
+-   Reveal the executable file in file manager. (#537)
 
 ### Fixed
 
-- Fixed the diff viewer crashes when the output is empty. (#517)
-- Fixed the background color of the diff viewer is black on Windows with the Dark Fusion theme. (#519 and #520)
-- Fixed the old problem URL is not restored when opening an old file with that option on. (#522)
-- Fixed the performance issue that makes the editor extremely slow when there are many lines. (cpeditor/QCodeEditor#27)
+-   Fixed the diff viewer crashes when the output is empty. (#517)
```

Comment:
Same unnecessary changes. This overwrites git blame, a quick way to find which feature in changelog was done by whom.

Reply:
It is better to use indents of width 4, more Markdown implementations support nested list with fixed-width indents, especially when the nested list is complicated.

I didn't think of `git blame` because I thought the PR number is much more useful. If you think it's important, I can revert these changes. However, I'm wondering whether the author of the changelog is always who implemented the feature. I usually use `git blame` to find the commit which introduces a bug, instead of to find the author of something.

Reply:
In fact, I didn't change any author:

![image](https://user-images.githubusercontent.com/30581822/89152467-c4fd4f00-d595-11ea-822a-10f0e07b0f07.png)

Reply:
In fact, I didn't change any author:

![image](https://user-images.githubusercontent.com/30581822/89152544-e9592b80-d595-11ea-8b34-be5b06ce17b7.png)

Do you think it's much easier to go to the PRs in the blame page instead of by the PR number in the brackets?

Reply:
It works offline. Since you have not changed anything it doesn't matter.